templates/.gitignore

```diff
@@ -0,0 +1,62 @@
+lib/
```

Comment:
Could be fancy and pull down https://github.com/github/gitignore/blob/master/Node.gitignore, then add our own special `lib/`, `es/`, and `build/` (or dist? whatever we call it) ignores.

Okay sorry that's an outright dumb idea